base62.go

```diff
@@ -1,81 +1,182 @@
 package ksuid
 
-import "strings"
+const (
+	// lexographic ordering (based on Unicode table) is 0-9A-Za-z
+	base62Characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+	zeroString       = "000000000000000000000000000"
+)
 
-// lexographic ordering (based on Unicode table) is 0-9A-Za-z
-var base62Characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+// Converts a base 62 byte into the number value that it represents.
+func base62Value(digit byte) byte {
+	switch {
+	case digit >= '0' && digit <= '9':
+		return digit - '0'
+	case digit >= 'A' && digit <= 'Z':
+		return 10 + (digit - 'A')
```

Comment:
Can you give these magic numbers a const? or maybe extract them into a global var since you're really talking about the offset of 'A' in base62Characters.